core/pki/pki.go

```diff
@@ -279,15 +285,14 @@ type MixDescriptor struct {
 	// to parameters.
 	Kaetzchen map[string]map[string]interface{} `json:",omitempty"`
 
-	// RegistrationHTTPAddresses is a slice of HTTP URLs used for Provider
-	// user registration. Providers of course may choose to set this to nil.
-	RegistrationHTTPAddresses []string
-
 	// Layer is the topology layer.
 	Layer uint8
 
 	// LoadWeight is the node's load balancing weight (unused).
 	LoadWeight uint8
+
+	// AuthenticationType is the authentication mechanism required
```

Comment:
I suggest we use this API comment space to mention the two above strings TrustOnFirstUseAuth and OutOfBandAuth. That way when we auto-generate the api docs we have something we can use.